src/documents/index.html.md.handlebars

```diff
@@ -2,41 +2,41 @@
-            <span role="presentation" class="idg-iconPractices">{{> practicesIcon }}</span>
-            <strong>Practices</strong></a> - putting the principles of inclusive design into action
-        </a>
-    </li>
-    <li>
-        <a href="/tools/UXWalkthroughs.html">
-            <span role="presentation" class="idg-iconTools">{{> toolsIcon }}</span>
-            <strong>Tools</strong></a> - detailed descriptions of tools for use in your inclusive design process
+	<li>
+		<span role="presentation" class="idg-iconDimensions">{{{getCategoryIcon "dimensions"}}}</span>
+        <strong>Recognize Diversity and Uniqueness</strong>
+		<p>As individuals spread out from the hypothetical average, the needs of individuals that are outliers, or at the margins, become ever more diverse. Most individuals stray from the average in some facet of their needs or goals. This means that a mass solution does not work well.</p>
+	</li>
+	<li>
+		<span role="presentation" class="idg-iconDimensions">{{{getCategoryIcon "dimensions"}}}</span>
+        <strong>Inclusive Process and Tools</strong>
+        <p>Inclusive design teams should be as diverse as possible and include individuals who have a lived experience of the users the designs are intended for.  This also respects the edict “nothing about us without us” without relegating people with disabilities to the role of subjects of research or token participants in design exercises.</p>
     </li>
-    <li>
-        <a href="/activities/MatchingGame.html">
-            <span role="presentation" class="idg-iconActivities">{{> activitiesIcon }}</span>
-            <strong>Activities</strong></a> - interesting and fun ways you can put it all together
-        </a>
+	<li>
+		<span role="presentation" class="idg-iconDimensions">{{{getCategoryIcon "dimensions"}}}</span>
+        <strong>Broader Beneficial Impact</strong>
+        <p>It is the responsibility of inclusive designers to be aware of the context and broader impact of any design and strive to effect a beneficial impact beyond the intended beneficiary of the design.</p>
     </li>
 </ul>
 </div>
 
-The Inclusive Design Guide focuses on the process of digital design. However, it can be applied to the design of
-<strong>services</strong>, the <strong>built environment</strong> and <strong>physical products</strong>, as well as processes like <strong>workshops</strong>,
-<strong>meetings</strong>, <strong>conferences</strong>, and even our daily interactions with one another. It can be used by anyone,
-and will remain a work in progress. The concepts and methods contained within are <strong>ever-evolving</strong>
-as we learn more about how to design inclusion into all that we do.
+<figure>
+[![Diagram showing the three dimensions of inclusive design and related insights and practices.](/images/InclusiveDesignDimensions.png)](/images/InclusiveDesignDimensions.png)
+<figcaption>
+*Figure 1:* The three dimensions of inclusive design
```

Comment:
Perhaps give a little more description of what this image shows or what the significance is.